_guides/integrations-3.md

```diff
@@ -1,30 +1,35 @@
 * Speeding up integrations by transferring the computing power needed away from client-side computer servers over to OpenLattice servers.
 * If OpenLattice is assisting with your integrations, security is enhanced by reducing the number of software platforms in the workflow process, via eliminating the need to utilize a third-party system (e.g., Dropbox) for transfer of individual files over to OpenLattice. 
 * One maintains full control over the data to be integrated from your server into the OpenLattice platform, including any changes over time.
 * Eliminating the need for client-side creation and storage of extra .csv files to send to OpenLattice.
 
-To give an overview of the process, a temporary table is copied over from a client-side server into OpenLattice's "Atlas" production server, which is set up on [AWS GovCloud](https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/whatis.html). This is done through a one-time vpn into a client-side computer by OpenLattice to set up the process, with your IT team's presence and support. OpenLattice then runs all data integrations on our side, on these temporary tables. For those that want continuous integrations (for instance, to add to and update CAD police data or behavioral health reports as they come in), a task can be set up on the client-side computer that runs at a set time period and over-writes the table that exists on	 the OpenLattice server, which is then integrated in parallel. These overwrites also ensure that the tables copied onto the OpenLattice side are temporary.
+To give an overview of the process, a temporary table is copied over from a client-side server into OpenLattice's "Atlas" production server, which is set up on [AWS GovCloud](https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/whatis.html).  OpenLattice then runs all data integrations on our side, on these temporary tables. For those that want continuous integrations (for instance, to add to and update CAD police data or behavioral health reports as they come in), the data transfer can be set up as a recurring task on the client-side computer to run at a set time period and over-write the table that exists on the OpenLattice server, which is then integrated in parallel. These overwrites also ensure that the tables copied onto the OpenLattice side are temporary.
+
+Your data would be stored in a private database accessible only to you - **please contact <a href="mailto:support@topenlattice.com">support@openlattice.com</a> for arrangements.** Later, one can still write your own integration script if desired and also put it onto OpenLattice servers to run automatically. A tutorial for this final step is being developed. 
```

Comment:
Super minor but should probably switch to either "Later, one can still write their own..." or "Later, you can still write your own..."